Dockerfile

```diff
@@ -2,12 +2,7 @@ ARG ALPINE_VERSION
 
 FROM alpine:${ALPINE_VERSION} AS build_erlang
 
-# Important!  Update this no-op ENV variable when this Dockerfile
```

Comment:
I think this can be removed if we just build with `no-cache` always?

Reply:
For other registries we do want to cache, so there is a point to keep it